apps/settings/test/marionette/app/regions/feedback.js

```diff
@@ -28,8 +28,7 @@ FeedbackPanel.Selectors = {
   'feedbackEmail': '#feedback-email',
   'feedbackEmailBar': '#feedback-emailbar',
   'sendFeedbackButton': '#feedback-send-btn',
-  'alertDialogMsg': '#feedback-alert-msg',
-  'alertDialogDoneBtn': '#feedback-done',
+  'alertDialogMsg': '#settings-alert-dialog.current .settings-dialog-message',
```

Comment:
For the previous failure, this selector is modified to match current alert dialog.